README.md

```diff
@@ -4,18 +4,628 @@
+```
+npm install --save-dev sequelize-cli
+```
+
+8. Configure Sequelize by creating `./src/config/sequelize.js`.
+
+Copy the following configurations in it.
+```
+const { URL } = require('url')
+const path = require('path')
+
+const DEFAULT = {
+  sequelize: config => {
+    let dialect = "postgres";
+    let host = "127.0.0.1";
+    let port = "5432";
+    let database = "actionhero";
+    let username = undefined;
+    let password = undefined;
+
+    // if your environment provides database information via a single JDBC-style URL like mysql://username:password@hostname:port/default_schema
+    if (process.env.DATABASE_URL) {
+      const parsed = new URL(process.env.DATABASE_URL);
+      if (parsed.username) {
+        username = parsed.username;
+      }
+      if (parsed.password) {
+        password = parsed.password;
+      }
+      if (parsed.hostname) {
+        host = parsed.hostname;
+      }
+      if (parsed.port) {
+        port = parsed.port;
+      }
+      if (parsed.pathname) {
+        database = parsed.pathname.substring(1);
+      }
+    }
+
+    return {
+      autoMigrate: true,
+      logging: false,
+      dialect: dialect,
+      port: parseInt(port),
+      database: database,
+      host: host,
+      username: username,
+      password: password,
+      models: [path.join(__dirname, "..", "models")],
+      migrations: [path.join(__dirname, "..", "migrations")]
+    };
+  }
+};
+
+module.exports.DEFAULT = DEFAULT;
+
+// for the sequelize CLI tool
+module.exports.development = DEFAULT.sequelize({
+  env: "development",
+  process: { env: "development" }
+});
+
+module.exports.staging = DEFAULT.sequelize({
+  env: "staging",
+  process: { env: "staging" }
+});
+
+module.exports.production = DEFAULT.sequelize({
+  env: "production",
+  process: { env: "production" }
+});
+```
+
+_Replace the database, username, host, passsword with your Postgresql configuration._
+
+
+9. To configure `sequelize-cli` use the above configuration and create a `.sequelizerc` file with following config. This file needs to be created in root folder of the project: `./.sequelizerc`.
+
+```
+const path = require('path');
+
+module.exports = {
+  'config': path.resolve('.', 'sequelize.js'),
+  'models-path': path.resolve('src', 'models'),
+  'seeders-path': path.resolve('src', 'seeders'),
+  'migrations-path': path.resolve('src', 'migrations')
+}
+```
+
+_The above configuration helps sequelize-cli know about the location to find other configurations and where to create models, seeders, migrations._
+
+
+10. Last step, to run the `sequelize-cli` from root folder, create a file called `sequelize.js` in the root folder and add the following:
+
+```
+const sequelizeConfig = require('./src/config/sequelize.js')
+
+const sequelizeConfigEnv = sequelizeConfig[process.env.NODE_ENV] || sequelizeConfig.DEFAULT
+module.exports = sequelizeConfigEnv.sequelize()
+```
+
+This in turn makes the CLI use configuration from `src/config/sequelize.js`.
+
```

Comment:
At some point, they will need to make a database `createdb actionhero` or similar